examples/coagents-starter-praisonai-agents/agent-py/research/demo.py

```
@@ -0,0 +1,187 @@
+                running=False,
+                active=False
+            )
+            
+            # Emit completion message
+            yield self._emit_message_event(outputs)
+            
+        except Exception as e:
+            error_msg = f"Error occurred: {str(e)}"
+            self.current_state["status"] = "error"
+            self.current_state["last_result"] = error_msg
+            
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)